### PR TITLE
Migrate to  node@4.4.2, eslint@2.7.0, babel-eslint@6.0.2 and eslint-p…

### PR DESCRIPTION
**Pull request by liubko**
…lugin-react@4.3.0

@playpauseandstop please take a look on the list of rules.
1. I'm not sure which rules should be in `base`, and which should be moved to `alternate`.
2. do you think we should add any of this rules? `"react/jsx-no-bind", "react/forbid-prop-types", "react/no-string-refs"`
3. what is `node_modules/eslint-config-ezhome` for?

### CODE REVIEW

**Comment by liubko** [comment]
Updated PR:
- use `eslint-plugin-import-order@2.1.2` plugin _(https://github.com/jfmengels/eslint-plugin-import-order)_

**Comment by playpauseandstop** [comment]
@liubko 

Thanks. Let discuss this tomorrow as well.

**Comment by liubko** [comment]
@playpauseandstop ^

File: package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "eslint-config-ezhome",
-  "version": "0.6.1",
-
+  "version": "1.0.0",
```
Comment (playpauseandstop) [inline package.json]: Please list this as `1.0.0-beta.1` version. I'd like to try it, but not sure that all necessary rules integrated yet.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+1.0.0 (2016-04-21)
```
Comment (playpauseandstop) [inline CHANGELOG.md]: > 1.0.0 (Not Released Yet)

**Comment by playpauseandstop** [comment]
LGTM

**Comment by playpauseandstop** [comment]
BTW, please check if it possible to update for even further versions. Thanks

**Comment by liubko** [comment]
> BTW, please check if it possible to update for even further versions. Thanks

Updated eslint-plugin-react to 5.1.1
- added new rule `eact/jsx-first-prop-new-line`
- added new rule `eact/jsx-no-target-blank`

File: README.md [hunk 3]
```diff
@@ -22,11 +22,11 @@ This package provides Ezhome's `.eslintrc` as an extensible shared config.
 
 ### Note on preferred ESLint versions
 
-As of `0.6.0` release, `eslint-config-ezhome` prefers:
+As of `1.0.0` release, `eslint-config-ezhome` prefers:
 
-* `eslint@1.10.3`
-* `babel-eslint@4.1.6`
-* `eslint-plugin-react@3.11.2`
+* `eslint@2.7.0`
```
Comment (playpauseandstop) [inline README.md:27]: 2.9.0

File: README.md [hunk 4]
```diff
@@ -22,11 +22,11 @@ This package provides Ezhome's `.eslintrc` as an extensible shared config.
...
+* `babel-eslint@6.0.2`
```
Comment (playpauseandstop) [inline README.md:28]: 6.0.4

File: README.md [hunk 5]
```diff
@@ -22,11 +22,11 @@ This package provides Ezhome's `.eslintrc` as an extensible shared config.
...
+* `eslint-plugin-react@4.3.0`
```
Comment (playpauseandstop) [inline README.md:29]: 5.1.1

File: package.json [hunk 6]
```diff
@@ -22,19 +21,17 @@
...
-    "babel-eslint": "4.1.6",
-    "eslint": "1.10.3",
-    "eslint-plugin-react": "3.11.2"
+    "babel-eslint": "6.0.4",
+    "eslint": "2.9.0",
+    "eslint-plugin-react": "5.1.1"
   },
   "eslintConfig": {
-    "extends": "ezhome/base"
+    "extends": "ezhome/base/index.js"
```
Comment (playpauseandstop) [inline package.json:33]: This should be reflected in README as well, if we need to extend exact modules, not packages as before

**Comment by playpauseandstop** [comment]
Please make an `v1.0.0-beta.1` tag and push it to GitHub, to create new version on NPM. Thanks

File: package.json (hunk 6 above)
Comment (liubko) [inline package.json:33]: @playpauseandstop not sure I understand you. this is to lint the repo itself. 
to extend from many we could do ```"extends": ["ezhome/base/index.js", "..."] or make changes to`ezhome/base/index.js` itself.

**Comment by liubko** [comment]
> Please make an v1.0.0-beta.1 tag and push it to GitHub, to create new version on NPM. Thanks

done

**Pull request merged**